man/chgpasswd.8.xml

```diff
@@ -131,29 +137,48 @@
 	  </para>
 	</listitem>
       </varlistentry>
-      <varlistentry condition="sha_crypt">
+      <varlistentry condition="bcrypt;sha_crypt;yescrypt">
 	<term><option>-s</option>, <option>--sha-rounds</option></term>
 	<listitem>
 	  <para>
 	    Use the specified number of rounds to encrypt the passwords.
 	  </para>
 	  <para>
-	    The value 0 means that the system will choose the default
-	    number of rounds for the crypt method (5000).
+	    You can only use this option with crypt method:
+	    <phrase condition="bcrypt">
+	    <replaceable>BCRYPT</replaceable></phrase>
+	    <phrase condition="sha_crypt">
+	    <replaceable>SHA256</replaceable>
+	    <replaceable>SHA512</replaceable></phrase>
+	    <phrase condition="yescrypt">
+	    <replaceable>YESCRYPT</replaceable></phrase>
 	  </para>
-	  <para>
-	    A minimal value of 1000 and a maximal value of 999,999,999
-	    will be enforced.
+	  <para condition="bcrypt">
+	    By default, the number of rounds for BCRYPT is defined by the
```

Comment:
I don't think you need `By default,` here. The default is explained below and is 13.

Reply:
There are two different defaults explained. Just as it has been before for the whole sha_crypt case. By default in this case means that these fields are parsed from login.defs if no `-s` option is applied.

The other default states which value will be used if no such fields exist in login.defs AND no `-s` option is supplied.

Reply:
Yes, I know that but the text that you wrote may not be understood like that. Perhaps changing to `The default number of rounds is 13`?